notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Is the error on line 71 understood?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>I'm not sure about the error, was only showing sometimes when re-running some blocks of the NB without restarting.</p><p>Anyway I decided to  just remove all this block. I don't want the notebook to write any files. It was something inherited from a pyirf example but I don't think is necessary to treat the results like this. </p>